src/webui/www/private/views/preferences.html

```diff
@@ -1280,6 +1280,18 @@
                     <input type="text" id="requestQueueSize" style="width: 15em;" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="storageType">QBT_TR(Storage Type (libtorrent >= 2.0; requires restart):)QBT_TR[CONTEXT=OptionsDialog]&nbsp;<a href="https://www.libtorrent.org/reference-Custom_Storage.html" target="_blank">(?)</a></label>
```

Comment:
This new option should be moved to just above "Disk queue size" as those has similar topics.